builds/switch/Makefile

```diff
@@ -0,0 +1,200 @@
+#---------------------------------------------------------------------------------
+.SUFFIXES:
+#---------------------------------------------------------------------------------
+
+ifeq ($(strip $(DEVKITPRO)),)
+$(error "Please set DEVKITPRO in your environment. export DEVKITPRO=<path to>/devkitpro")
+endif
+
+ifeq ($(strip $(TOOLCHAIN_DIR)),)
+$(error "Please point TOOLCHAIN_DIR to your switch portlibs directory")
+endif
+
+ifeq ($(strip $(LIBLCF_DIR)),)
+$(warning "LIBLCF_DIR not set. Ignore this warning when liblcf is in TOOLCHAIN_DIR")
+LIBLCF_DIR := $(TOOLCHAIN_DIR)
+endif
+
+TOPDIR ?= $(CURDIR)
+include $(DEVKITPRO)/libnx/switch_rules
+
+#---------------------------------------------------------------------------------
+# TARGET is the name of the output
+# BUILD is the directory where object files & intermediate files will be placed
+# SOURCES is a list of directories containing source code
+# DATA is a list of directories containing data files
+# INCLUDES is a list of directories containing header files
+# EXEFS_SRC is the optional input directory containing data copied into exefs, if anything this normally should only contain "main.npdm".
+# 
+# NO_ICON: if set to anything, do not use icon.
+# NO_NACP: if set to anything, no .nacp file is generated.
+# APP_TITLE is the name of the app stored in the .nacp file (Optional)
+# APP_AUTHOR is the author of the app stored in the .nacp file (Optional)
+# APP_VERSION is the version of the app stored in the .nacp file (Optional)
+# APP_TITLEID is the titleID of the app stored in the .nacp file (Optional)
+# ICON is the filename of the icon (.jpg), relative to the project folder.
+#   If not set, it attempts to use one of the following (in this order):
+#     - <Project name>.jpg
+#     - icon.jpg
+#     - <libnx folder>/default_icon.jpg
+#---------------------------------------------------------------------------------
+TARGET		:=	easyrpg-player
+BUILD		:=	build
+SOURCES		:=	../../src
+DATA		:=	data
+INCLUDES	:=	../../src
+EXEFS_SRC	:=	exefs_src
+ICON		:= icon.jpg
+
+#---------------------------------------------------------------------------------
+# options for code generation
+#---------------------------------------------------------------------------------
+ARCH	:=	-march=armv8-a -mtune=cortex-a57 -mtp=soft -fPIE
+
+CFLAGS	:= -g -Wall -O2 -ffast-math -ffunction-sections \
+			$(ARCH) $(DEFINES)
+
+CFLAGS	+=	$(INCLUDE) -DSWITCH -DHAVE_LIBSAMPLERATE -DHAVE_LIBSNDFILE \
+			-DHAVE_OGGVORBIS -DHAVE_MPG123 -DSUPPORT_AUDIO \
+			-DHAVE_WILDMIDI -DWANT_FMMIDI -DWANT_FASTWAV \
+			-I$(TOOLCHAIN_DIR)/include \
+			-I$(LIBLCF_DIR)/include/liblcf \
+			-I$(TOOLCHAIN_DIR)/include/pixman-1
```

Comment:
Please also add the other libs I provided:
HAVE_OPUS
HAVE_XMP
HAVE_FREETYPE

with appropriate deps (not sure if all filenames are correct)
-lxmplite -lopusfile -lopus -lfreetype

You can test if XMP (MOD/Tracker music) works e.g. with https://rpgmaker.net/games/4145/downloads/3594/ (Title music)